### PR TITLE
Blog Post: Long-Term Support For Alex Gleason

### PR DESCRIPTION
**Pull request by Arvin21M**
**Build Preview:**
- [Long-Term Support For Alex Gleason](https://os-website-git-lts-for-alex-gleason-open-sats.vercel.app/blog/alex-gleason-receives-lts-grant)

**Social Preview:**
- image

### CODE REVIEW

**Comment by alexgleason** [comment]
Thank you!!

**Comment by dergigi** [comment]
Looks good to me! Please do a quick sanity check if you have a minute @OpenSats/volunteers 🙏🧡

**Comment by btcschellingpt** [comment]
Reads well .. only thing I can see is that the "fediverse" links to activity.pub .. is that the best reference for Fediverse?

File: data/blog/alex-gleason-receives-lts-grant.mdx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+Alex is most recently known for launching [Ditto](https://soapbox.pub/ditto/), a
+decentralized, self-hosted social media server that prioritizes user control and
+community building, featuring a built-in nostr client and relay, compatible with
+Mastodon apps. He presented [Ditto at the Nostriga
+conference](https://www.youtube.com/watch?v=iZyqVGHjDGo) in August 2024.
+
+Beyond Ditto, Alex has made substantial contributions through several other key
+projects, including the [Mostr
+Bridge](https://soapbox.pub/blog/mostr-fediverse-nostr-bridge/), which connects
+nostr with the [Fediverse](https://activitypub.rocks/);
```
Comment (Bayernatoor) [inline data/blog/alex-gleason-receives-lts-grant.mdx:24]: The Fediverse link seems incorrect? @Arvin21M, unless you meant to link out to acitivitypub.rocks ?

**Review by Bayernatoor — COMMENTED** [review]
Just the one potential inccorect link. Besides that it looks good and everything else works as it should. Met Alex in Nostriga, very happy to see him getting a grant. Ditto seems really promising.

**Comment by Arvin21M** [comment]
@Bayernatoor 
> The Fediverse link seems incorrect? @Arvin21M, unless you meant to link out to activitypub.rocks ?

@btcschellingpt 
> Only thing I can see is that the "fediverse" links to activity.pub .. is that the best reference for Fediverse?

Fediverse is essentially a general term coined several years ago to describe how several social networks can communicate with one another. ActivityPub is one these bridges for cross communication. 

Since there is no "official" write up, as an alternative, we can include the [Wikipedia description](https://en.wikipedia.org/wiki/Fediverse) for it, or remove the URL all together.

**Comment by Arvin21M** [comment]
Swapped the link in question.

**Review by btcschellingpt — APPROVED** [review]
Good to freakin' go

**Review by dergigi — APPROVED** [review]

**Pull request merged**